tests/jax/test_custom_call_compute.py

```diff
@@ -432,7 +432,7 @@ def ref_act_lu(inputs):
     def primitive_func(self, inputs):
         return jnp.mean(activation_lu(inputs, activation_type=self.activation_type))
 
-    @pytest.mark.parametrize("shape", [(32, 1, 64), (64, 1, 256)])
+    @pytest.mark.parametrize("shape", [(32, 1, 64), (16, 64, 1, 256)])
```

Comment:
Why the change in shape?

Reply:
Hi, I found out that we only had three-dimensional input cases, so I changed it to have a case with more than 3 dimensions.